pkg/cloud/openstack/assets/rbac/clusterrolebinding.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: openstack-cloud-controller-manager
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+roleRef:
+  kind: ClusterRole
+  name: cloud-controller-manager
+subjects:
+- kind: ServiceAccount
+  namespace: openshift-cloud-controller-manager
+  name: openstack-cloud-controller-manager
```

```suggestion
  name: cloud-controller-manager
```

This will be reused by all providers.